ui/lib/core/addon/components/key-value-header.hbs

```diff
@@ -3,25 +3,17 @@
   SPDX-License-Identifier: BUSL-1.1
 ~}}
 
-<nav class="key-value-header breadcrumb" aria-label="breadcrumbs">
-  <ul>
-    {{yield}}
-    {{#each this.secretPath as |path index|}}
-      <li class={{if (is-active-route path.path path.model isExact=true) "is-active"}}>
-        <span class="sep">&#x0002f;</span>
-        {{#if this.linkToPaths}}
-          <LinkTo
-            @route={{path.path}}
-            @model={{path.model}}
-            data-test-secret-root-link={{if (eq index 0) true false}}
-            data-test-secret-breadcrumb={{path.text}}
-          >
-            {{path.text}}
-          </LinkTo>
-        {{else}}
-          <span>{{path.text}}</span>
-        {{/if}}
-      </li>
-    {{/each}}
-  </ul>
-</nav>
\ No newline at end of file
+<Hds::Breadcrumb>
```

Comment:
This component should ultimately be replaced since all of the getters are overly robust, but felt outside of the scope of this PR. As a stop-gap I just replaced instances that `{{yield}}` was used